src/status_im2/common/contacts/views.cljs

```diff
@@ -0,0 +1,77 @@
+(ns status-im2.common.contacts.views
+  (:require [status-im2.common.contacts.style :as style]
+            [i18n.i18n :as i18n]
+            [utils.debounce :as debounce]
+            [utils.re-frame :as rf]
+            [react-native.core :as rn]
+            [quo2.core :as quo2]
+            [quo2.foundations.colors :as colors]))
+
+(def add-contact-img (js/require "../resources/images/ui/add-contact.png"))
```

Comment:
we have `status-im.react-native.resources` for images

Reply:
I did notice `react-native.resources`, and was about to use that, but two things nudged me in the other direction:
1. `status-im2/contexts/communities/overview/view.cljs`, which is new code, didn't use `react-native.resources`, so I followed that pattern.
2. The new guidelines didn't mention anything about it.

So, I'd suggest:
1. Update `status-im2/contexts/communities/overview/view.cljs` to use `react-native.resources`
2. Update new-guidelines to recommend using `react-native.resources` for images

Just updating `status-im2/contexts/communities/overview/view.cljs` is probably enough, imo.

Thanks Andrey!

Reply:
@erikseppanen, since you're touching on the topic of the guidelines not mentioning about `status-im.react-native.resources`, I think this is a good example of something that we could avoid adding to the guidelines IF we do some refactoring (in a separate PR).

So instead of updating the guidelines about using `status-im.react-native.resources`, a stronger *convention* would be to replace all instances of `js/require` that are used for loading resources. This way, we reduce the chance of devs getting confused by two competing code standards/patterns.

It's similar to what @vkjr did recently to replace all usages of the old `>evt` with `utils.re-frame/dispatch`. Instead of documenting how to do something, he fixed the problem from the root.

Reply:
@ilmotta Yup, I'm leaning in that direction too. A good metaphor might be: if developers just see one well-trodden path in the codebase, we usually don't need to add a guardrail (or paint stripes, etc.).